cmd/credentialUtil.go

```diff
@@ -186,7 +229,8 @@ func getBlobFSCredentialType(ctx context.Context, blobResourceURL string, standa
 	if name != "" && key != "" { // TODO: To remove, use for internal testing, SharedKey should not be supported from commandline
 		return common.ECredentialType.SharedKey(), nil
 	} else {
-		return common.ECredentialType.Unknown(), errors.New("OAuth token, SAS token, or shared key should be provided for Blob FS")
+		/* We dont return error here. The caller can either fail or attempt auto-login */
+		return common.ECredentialType.Unknown(), nil
```

Comment:
This is also called in cmd/make.go-- To avoid breaking compatibility, perhaps we should return an additional flag, or make the error static and check for it. Alternatively, we can implement auto-login in create as well (Which probably has the higher engineering cost of the two options here)

For instance, `var blobFSNoAuthError = errors.New("Not enough authentication detail for blobFS")` (not withholding the actual phrasing here), then when we are attempting auto-login, we should check if the resulting error is this one.

Reply:
I noticed you did change make.go to appropriately handle this-- I do think though, that the marked error would be best, that way, it's not a bit of "mental overhead" when we write new code with this.

Reply:
AzCopy errors are not helpful to compare. I took an easy way out I guess. Changin this.